netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+  publish = "_public/"
+
```

Comment:
lets delete netlify.toml we won't publish anything to netlify from this repo anyways